front/components/assistant/conversation/SidebarMenu.tsx

```diff
@@ -228,25 +228,44 @@ export function AssistantSidebarMenu({ owner }: AssistantSidebarMenuProps) {
               </div>
             ) : (
               <div className={classNames("flex justify-end gap-2 pt-2")}>
+                <Button
+                  label="New"
+                  icon={ChatBubbleBottomCenterPlusIcon}
+                  className="shrink"
+                  tooltip="Create a new conversation"
+                  href={`/w/${owner.sId}/assistant/new`}
+                  onClick={() => {
+                    setSidebarOpen(false);
+                    const { cId } = router.query;
+                    const isNewConversation =
+                      router.pathname === "/w/[wId]/assistant/[cId]" &&
+                      typeof cId === "string" &&
+                      cId === "new";
```

Comment:
`nit`: that's a bit redundant

Reply:
Not many other options I'm afraid..

Reply:
That's not from the PR (pre-existing code)